### PR TITLE
Test for empty input/column names

### PR DESCRIPTION
**Pull request by TimNooren**
Closes https://github.com/blockchain-etl/ethereum-etl-airflow/issues/458.

### CODE REVIEW

File: tests/test_validate_json.py [hunk 1]
```diff
@@ -83,3 +83,23 @@ def test_table_name_is_correct(filename):
...
     expected_table_name = basename.replace('.json', '')
     assert expected_table_name == table_name
+
+
+@pytest.mark.parametrize("filename", all_files)
+def test_no_empty_event_input_names(filename):
+    json_data = load_json_file(filename)
+    abi = json_data["parser"]["abi"]
+    for i, _input in enumerate(abi["inputs"]):
+        assert _input["name"] != "", \
```
Comment (charlielewisme) [inline tests/test_validate_json.py:93]: Does not catch the condition where `"name": null` in the json.

Suggestion:
```
assert _input["name"] not in (None, ""), \
```

Likewise line 103

**Review by charlielewisme — COMMENTED** [review]
Looks good but just want to check something.

File: tests/test_validate_json.py (hunk 1 above)
Comment (TimNooren) [reply tests/test_validate_json.py]: My bad, I had another look and realised we already use jsonschema for validation, which is probably a better way to this to begin with.

**Review by charlielewisme — APPROVED** [review]
Nice.

**Review by charlielewisme — APPROVED** [review]
Added descriptions are fine. LGTM.

**Pull request merged**